hack/infra/sources/infra_init.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+## FileName: install_containerd.sh
```

Comment:
The filename of this script is different from it in this comment.

Reply:
fixed